src/tests/system/tests/test_tools.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tools Tests.
+
+Tests pertaining to command line tools, some tools will have their own file.
```

Comment:
Out of curiosity how will this be determined which tools will go in their own file and which in `test_tools.py` ?

Reply:
It usually boils down to the complexity and how many tests they are and whether it makes sense or not, but there is nothing concrete.